js/index.js

```diff
@@ -8,35 +8,130 @@ const siteContent = {
+navLinks[0].textContent = siteContent["nav"]["nav-item-1"];
+navLinks[1].textContent = siteContent["nav"]["nav-item-2"];
+navLinks[2].textContent = siteContent["nav"]["nav-item-3"];
+navLinks[3].textContent = siteContent["nav"]["nav-item-4"];
+navLinks[4].textContent = siteContent["nav"]["nav-item-5"];
+navLinks[5].textContent = siteContent["nav"]["nav-item-6"];
+navLinks[6].setAttribute("href", "#");
+navLinks[7].setAttribute("href", "#");
+
+navLinks.forEach(a => (a.style.color = "green"));
+
+let ctaHeader = document.querySelector(".cta-text h1");
+let ctaButton = document.querySelector(".cta-text button");
+let ctaImg = document.getElementById("cta-img");
+ctaHeader.textContent = "DOM";
+ctaHeader.appendChild(document.createElement("br"));
+ctaHeader.appendChild(document.createTextNode("Is"));
+ctaHeader.appendChild(document.createElement("br"));
+ctaHeader.appendChild(document.createTextNode("Awesome"));
+ctaButton.textContent = siteContent["cta"]["button"];
+ctaImg.setAttribute("src", siteContent["cta"]["img-src"]);
+
+let mainContentH4 = document.querySelectorAll(".text-content h4");
+let mainContentP = document.querySelectorAll(".text-content p");
+mainContentH4[0].textContent = siteContent["main-content"]["features-h4"];
+mainContentP[0].textContent = siteContent["main-content"]["features-content"];
+mainContentH4[1].textContent = siteContent["main-content"]["about-h4"];
+mainContentP[1].textContent = siteContent["main-content"]["about-content"];
+
+let middleImg = document.getElementById("middle-img");
+middleImg.setAttribute("src", siteContent["main-content"]["middle-img-src"]);
+
+mainContentH4[2].textContent = siteContent["main-content"]["services-h4"];
+mainContentP[2].textContent = siteContent["main-content"]["services-content"];
+mainContentH4[3].textContent = siteContent["main-content"]["product-h4"];
+mainContentP[3].textContent = siteContent["main-content"]["product-content"];
+mainContentH4[4].textContent = siteContent["main-content"]["vision-h4"];
+mainContentP[4].textContent = siteContent["main-content"]["vision-content"];
+
+//stretch
+let newContactP = document.createElement("p");
+let contact = document.querySelector(".contact");
+contact
+  .appendChild(newContactP)
+  .appendChild(document.createElement("button")).textContent = "Click!";
+contact
+  .appendChild(document.createElement("div"))
+  .appendChild(document.createElement("div"));
+
+let contactDiv = document.querySelector(".contact div");
+contactDiv.style.cssText =
+  "width: 400px; height: 400px; position: relative; background: yellow";
+let contactInnerDiv = document.querySelector(".contact div div");
+contactInnerDiv.style.cssText =
+  "width: 50px; height: 50px; position: absolute; background-color: red;";
+contactInnerDiv.classList.add("animation");
+let contactButton = document.querySelector(".contact p button");
+contactButton.onclick = function() {
+  var elem = contactInnerDiv;
+  var pos = 0;
+  var id = setInterval(frame, 5);
+  function frame() {
```

Comment:
This is neat, but you can do this much easier with GreenSock or CSS animations. If you have the opportunity to, I really recommend trying GreenSock on the project today https://greensock.com/ .